python/pyarrow/tests/test_table.py

```diff
@@ -972,65 +977,67 @@ def check_tensors(tensor, expected_tensor, type, size):
     assert tensor.strides == expected_tensor.strides
 
 
-@pytest.mark.parametrize('typ', [
-    np.uint8, np.uint16, np.uint32, np.uint64,
-    np.int8, np.int16, np.int32, np.int64,
-    np.float32, np.float64,
-])
```

Comment:
If we would want to keep this parametrization, could easily use strings instead (and then inside the function convert that with `np.dtype(..)` or so). 
But no strong opinion, iterating over the options inside the test is also fine.